packages/taquito/src/operations/transaction-operation.ts

```diff
@@ -66,7 +66,8 @@ export class TransactionOperation extends Operation
     return this.params.storage_limit;
   }
 
-  private sumProp(arr: any[], prop: string) {
+  // CHECK this removal of an "any array"
```

Comment:
looks good, can you remove the comment and add the missing operations to `MergedOperationResult`?